external/fv3fit/fv3fit/train.py

```diff
@@ -88,47 +78,55 @@ def _get_model(config: ModelTrainingConfig) -> Estimator:
             optimizer=get_optimizer(config.hyperparameters),
             kernel_regularizer=get_regularizer(config.hyperparameters),
             checkpoint_path=checkpoint_path,
-            fit_kwargs=fit_kwargs,
             **config.hyperparameters,
         )
     else:
         raise NotImplementedError(f"Model type {config.model_type} is not implemented.")
 
 
-def _keras_fit_kwargs(config: ModelTrainingConfig) -> dict:
-    # extra args specific to keras training
-    fit_kwargs = config.hyperparameters.pop("fit_kwargs", {})
-    fit_kwargs["validation_dataset"] = validation_dataset(data_path, train_config)
-    return fit_kwargs
-
-
 if __name__ == "__main__":
     logging.basicConfig(level=logging.INFO)
     parser = get_parser()
     args = parser.parse_args()
     data_path = parse_data_path(args.data_path)
-    train_config = ModelTrainingConfig.load(args.config_file)
-    train_config.data_path = args.data_path
-
-    if args.timesteps_file:
-        with open(args.timesteps_file, "r") as f:
-            timesteps = yaml.safe_load(f)
-        train_config.batch_kwargs["timesteps"] = timesteps
-        train_config.timesteps_source = "timesteps_file"
-
-    if args.validation_timesteps_file:
-        with open(args.validation_timesteps_file, "r") as f:
-            val_timesteps = yaml.safe_load(f)
-        train_config.validation_timesteps = val_timesteps
-
-    train_config.dump(args.output_data_path)
+    (
+        legacy_config,
+        train_config,
+        train_data_config,
+        val_data_config,
+    ) = fv3fit._shared.config.load_configs(
+        args.config_file,
+        data_path=data_path,
+        output_data_path=args.output_data_path,
+        timesteps_file=args.timesteps_file,
+        validation_timesteps_file=args.validation_timesteps_file,
+    )
     set_random_seed(train_config.random_seed)
 
-    batches = load_data_sequence(data_path, train_config)
+    train_config.dump(os.path.join(args.output_data_path, "train.yaml"))
```

Comment:
```suggestion
    train_config.dump(os.path.join(args.output_data_path, "train.yml"))
```

Reply:
I changed these new ones to use `.yaml` since that's what's [officially recommended](https://yaml.org/faq.html).